app/views/admin/_auctions_subnav.html.erb

```diff
@@ -7,7 +7,11 @@
       class: view_model.auctions_nav_class %></li>
     <li><%= link_to t('links_and_buttons.auctions.needs_attention'),
       admin_auctions_needs_attention_path,
-      class: view_model.needs_attention_auctions_nav_class %></li>
+      class: view_model.needs_attention_auctions_nav_class %>
+      <% if view_model.needs_attention_auctions_count > 0 %>
```

Comment:
let's add this logic to the view model so we don't need to put logic in the view (use the `components/null` pattern)